archive/sensitivity.lean

```diff
@@ -278,7 +278,7 @@ lemma f_squared : ∀ v : V n, (f n) (f n v) = (n : ℝ) • v :=
 begin
   induction n with n IH; intro,
   { simpa only [nat.cast_zero, zero_smul] },
-  { cases v, simp [f_succ_apply, IH, add_smul], abel }
+  { cases v, simp [f_succ_apply, IH, add_smul, add_assoc], abel }
```

Comment:
Is `add_assoc` no longer simp?

Reply:
It was never supposed to be. It was officially removed as a simp lemma at the same time as `mul_assoc`, but due to a bug in the core version of `to_additive` it was getting marked as a simp lemma anyway.

Reply:
Oh, I didn't notice.  I only remember the `add_comm`/`mul_comm` story, where clearly neither should be simp lemmas.  Is there a good reason `*_assoc` should not be simp?

Reply:
I'm not totally clear on the reasons for removing these in the first place, but I think that consistency is good. Quite clearly even `add_comm` is okay for simp because it has explicit support for AC rewriting. Maybe these lemmas should be added to a simp set like `simp with ac`?